concordium-sdk/src/main/java/com/concordium/sdk/requests/Range.java

```diff
@@ -0,0 +1,65 @@
+package com.concordium.sdk.requests;
+
+import lombok.*;
+
+import java.util.Optional;
+
+/**
+ * A range parameter for queries in {@link com.concordium.sdk.ClientV2}.
+ * @param <T> type of the values in the range.
+ */
+@EqualsAndHashCode
+@Builder(access = AccessLevel.PRIVATE)
+@ToString
+public class Range<T> {
```

Comment:
I've taken a look at it, and it does not seem to allow for the range to be unbounded in any direction. Instead of `Range` parameter I can alternatively use `AbsoluteBlockHeight` as `from` and `to` parameter and overload the method to allow for unbounded ranges?

Reply:
Ah, I did not consider that unbounded part. Let's just keep this then.